### PR TITLE
Adding textField color, size, and font adjustment example

### PR DESCRIPTION
**Pull request by langsmith**
This pr adds an example of using runtime styling to adjust the various `textField` properties of a `SymbolLayer`. This example shows toggling between 2 different colors, text sizes, and fonts.

ezgif com-resize 2

### CODE REVIEW

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+ */
+public class TextFieldFormattingActivity extends AppCompatActivity {
+
+  private final String[] textFonts = new String[] {"Roboto Black", "Arial Unicode MS Regular"};
+  private final Float[] textSizes = new Float[] {25f, 13f};
+  private final String[] textColors = new String[] {"#00FF08", "#ffd43a"};
+  private MapView mapView;
+  private boolean fontChanged = false;
+  private boolean sizeChanged = false;
+  private boolean colorChanged = false;
```
Comment (tobrun) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: no need to assign false to field declaration, they are already false.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.textSize;
+
+/**
+ * Use runtime styling to adjust the font, size, and color of SymbolLayer text fields.
+ */
+public class TextFieldFormattingActivity extends AppCompatActivity {
+
+  private final String[] textFonts = new String[] {"Roboto Black", "Arial Unicode MS Regular"};
+  private final Float[] textSizes = new Float[] {25f, 13f};
+  private final String[] textColors = new String[] {"#00FF08", "#ffd43a"};
```
Comment (tobrun) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: above fields can be made static

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            });
+
+            FloatingActionButton textColorFab = findViewById(R.id.fab_toggle_text_color);
+            textColorFab.setOnClickListener(new View.OnClickListener() {
+              @Override
+              public void onClick(View view) {
+                adjustText(style, textColor(colorChanged ? textColors[1] : textColors[0]));
+                colorChanged = !colorChanged;
+              }
+            });
```
Comment (tobrun) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: can we refactor above statements, they contain similar functionality.
Additionally assigning and setting the click listener can be done in one line instead.

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  /**
+   * Retrieve a specific layer and adjust a property of its text field.
+   *
+   * @param loadedMapStyle a fully loaded map {@link Style} object
+   * @param propertyValue the {@link com.mapbox.mapboxsdk.style.layers.PropertyFactory} to adjust for
+   *                      the specific layer's text field.
+   */
+  private void adjustText(@NonNull Style loadedMapStyle, @NonNull PropertyValue propertyValue) {
+    for (Layer singleMapLayer : loadedMapStyle.getLayers()) {
+      if (singleMapLayer.getId().contains("water-") && !singleMapLayer.getId().equals("water-shadow")) {
```
Comment (tobrun) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: constants?

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java (hunk 1 above)
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: fixed

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java (hunk 2 above)
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: fixed

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java (hunk 4 above)
Comment (langsmith) [reply MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TextFieldFormattingActivity.java]: fixed

**Pull request merged**